src/main/java/org/jabref/gui/preferences/appearance/AppearanceTab.fxml

```diff
@@ -47,4 +49,9 @@
             </tooltip>
         </Button>
     </HBox>
+    <Label fx:id="restartJabRefHint" styleClass="text" text="${'\nRestarting JabRef may be required to display changes in theme preferences.'}">
+        <font>
+            <Font name="Italic" size="13.0"/>
+        </font>
+    </Label>
```

Comment:
Why just fixed the restart for theme change issue a few months back. This would be a major regression.